lib/bovine/staticinventory.py

```diff
@@ -6,7 +6,7 @@
 ##########################
 
 
-class Inventory:
+class GatherStaticInventory:
```

Comment:
We will want to rename this.  GatherStaticInventory sounds more like a method name (e.g. VerbObject).  Class names should be a 'class' of things... we will probably want the Class name to either be
- Inventory
of
- StaticInventory

I suppose StaticInventory could be a subclass of Inventory.  

Don't make any changes for now though.  This looks good for initial tests.